### PR TITLE
Disable refactor for jerome

### PR DESCRIPTION
**Pull request by JayRome1505**
UI Changes

Navbar
- Now compatible with mobile (hopefully <3)
-
Login and sign up
- Changed image
- theme change
- removed unused buttons

Home
- Mobile page doesnt look ugly anymore yay

scanImage
- Looks aight on mobile now, fits on page properly. Results are kinda??? tho idk wasnt a me thing

Screenshot_2023-10-29-18-37-57-690_com android chrome
Screenshot_2023-10-29-18-37-34-807_com android chrome
Screenshot 2023-10-29 at 6 34 26 pm
Screenshot 2023-10-29 at 6 34 36 pm
Screenshot 2023-10-29 at 6 34 50 pm
Screenshot 2023-10-29 at 6 34 58 pm
Screenshot 2023-10-29 at 6 35 56 pm
Screenshot 2023-10-29 at 6 42 29 pm

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -56,9 +56,10 @@ const App = () => {
         <Route path="/PICResult7" element={<PICResult7 />} />
         <Route path="/YellowBinResult" element={<YellowBinResult />} />
         <Route path="/RedBinResult" element={<RedBinResult />} />
+        <Route path="/ScanImage" element={<ScanImage />} />
 
         <Route path="/Account" element={<PrivateRoute><Account /></PrivateRoute>}/>
-        <Route path="/ScanImage" element={<PrivateRoute><ScanImage /></PrivateRoute>}/>
+        {/* <Route path="/ScanImage" element={<PrivateRoute><ScanImage /></PrivateRoute>}/> */}
```
Comment (JasmineBautista) [inline client/src/App.js]: not sure if you wanna merge this to main

Comment (JayRome1505) [reply client/src/App.js]: How do i not do that

**Review by swolbean — CHANGES_REQUESTED** [review]
image
Idk why this error popped up on account but could this be fixed first? 

image
phone navbar seems default

**Review by swolbean — APPROVED** [review]
Cool done aAA

**Pull request merged**